Mono.Debugging/Mono.Debugging.Evaluation/ObjectValueAdaptor.cs

```diff
@@ -287,6 +287,7 @@ public virtual void OnBusyStateChanged (BusyStateEventArgs e)
 		public abstract bool IsEnum (EvaluationContext ctx, object val);
 		public abstract bool IsValueType (object type);
 		public abstract bool IsClass (EvaluationContext ctx, object type);
+		public abstract bool IsDelayedType (EvaluationContext ctx, object type);
```

Comment:
@DavidKarlas @jstedfast are we API stable? If so, we need to avoid adding new abstract members (use virtual) and changing abstract to non-abstract here.